src/controller/securemgr/authorizer/authorizer.go

```diff
@@ -88,15 +91,15 @@ func Init(rbacRulePath string) {
 	}
 	rbacPolicyFilePath = rbacRulePath + "/" + rbacPolicyFileName
 	if _, err := os.Stat(rbacPolicyFilePath); err != nil {
-		err = ioutil.WriteFile(rbacPolicyFilePath, []byte(policy_template), 0666)
+		err = ioutil.WriteFile(rbacPolicyFilePath, []byte(policyTemplate), 0666)
```

Comment:
Sorry for other topic...
How about changing the file permission from 0666 to 0664?
I think granting write permission to any other users is unnecessary.

Reply:
> 
> 
> Sorry for other topic...
> How about changing the file permission from 0666 to 0664?
> I think granting write permission to any other users is unnecessary.

@t25kim Done! Thank you for your comment!